stable/prometheus-postgres-exporter/templates/deployment.yaml

```diff
@@ -48,6 +48,9 @@ spec:
           - {{ .Values.config.excludeDatabases | join "," }}
           {{- end }}
           {{- end }}
+{{- if .Values.config.extraArgs }}
+{{ toYaml .Values.config.extraArgs | indent 12 }}
```

Comment:
Shouldn't that be indent 10?

Reply:
you can template it out and test it, afaik, 12 is correct 😄